cypress/integration/leaderboard_spec.js

```diff
@@ -0,0 +1,37 @@
+const url = Cypress.config().baseUrl;
+
+let score;
+
+describe('Leaderboard', () => {
+    describe('on logging in for the first time', () => {
+        beforeEach(() => {
+            cy.login('unicorn')
+            .then(() => cy.getPlayers());
+        });
+
+        it('displays the username with a score of zero', () => {
+            cy.get('[data-cy="player-username"]').contains('unicorn');
+            score = cy.get('[data-cy="player-score"]').first().then($score => {
+                score = $score.text();
```

Comment:
Is this test doing anything?

Reply:
it checks if the player's username is there, and gets the previous score (before one game is completed), but I should change the name, sorry